api/v1beta1/webhook.go

```diff
@@ -282,7 +282,7 @@ type defaulter interface {
 
 func isSuperUser(ctx context.Context, user authenticationv1.UserInfo) bool {
 	operatorUser, _ := operatorUserFromContext(ctx)
-	return operatorUser == user.Username || util.ContainsValue("system:masters", user.Groups)
+	return operatorUser == user.Username || util.ContainsValue("system:masters", user.Groups) || util.ContainsValue("system:serviceaccounts:kube-system", user.Groups)
```

Comment:
what is this for ?

Reply:
see https://github.com/bestchains/fabric-operator/pull/62#issue-1533179564
When kubernetes gc, user
```
&UserInfo{
	Username:system:serviceaccount:kube-system:generic-garbage-collector,
	UID:201cb729-fab1-4559-bbb7-ef2ee367deed, #random
	Groups:[system:serviceaccounts system:serviceaccounts:kube-system system:authenticated],
	Extra:map[string]ExtraValue{},
}
```
will delete resource, we should let these requests pass.